.eslintrc.cjs

```diff
@@ -16,5 +16,6 @@ module.exports = {
       'warn',
       { allowConstantExport: true },
     ],
+    'no-unused-vaars': false,
```

Comment:
Are there any unused variables that can't be deleted?

If not, then remove them and this rule as well.